gammapy/spectrum/dataset.py

```diff
@@ -676,6 +677,96 @@ def read(cls, filename):
             "To read from an OGIP fits file use SpectrumDatasetOnOff.from_ogip_files."
         )
 
+    def stack(self, other):
+        """Stack this dataset with another one.
+
+        Safe mask is applied to compute the stacked counts vector.
+        Counts outside each dataset safe mask are lost.
+
```

Comment:
Maybe add a sentence about the method that is used to stack the alpha (and average alpha)?

Reply:
Done.